FirebaseAnalytics.podspec

```diff
@@ -0,0 +1,50 @@
+Pod::Spec.new do |s|
```

Comment:
Does it have something to do with the fact the JSON version of 9.1.0 is already staged on specs staging? Thoughts?
Update: I have locally converted the GoogleAppMeasurement podspec and am seeing the same issue.
Update: I ran pod lib lint because that what's I see we do in [analytics.yml](../../blob/master/.github/workflows/analytics.yml#L38).

Reply:
Passing `--verbose` did not surface anything insightful.

I think this may not be specific to my changes. I'm seeing this behavior on `master` too:
```
➜  firebase-ios-sdk git:(master) 
‣ git checkout master                                                                                                                                                                                                                               
Already on 'master'
Your branch is up to date with 'origin/master'.
➜  firebase-ios-sdk git:(master) 
‣ pod cache clean --all                                                                                                                                                                                                                             
➜  firebase-ios-sdk git:(master) 
‣ bundle exec pod lib lint \                                                                                                                                                                                                                        
    --sources=https://github.com/firebase/SpecsDev.git,https://github.com/firebase/SpecsStaging.git,https://cdn.cocoapods.org/ \
    --include-podspecs=\{FirebaseCore.podspec,FirebaseCoreDiagnostics.podspec,FirebaseCoreInternal.podspec,FirebaseInstallations.podspec\} \
    --analyze FirebaseAnalytics.podspec.json \
    --platforms=macos \
    --skip-import-validation \
    --no-clean

 -> FirebaseAnalytics (9.1.0)
    - ERROR | [FirebaseAnalytics/AdIdSupport,FirebaseAnalytics/WithoutAdIdSupport] file patterns: The `vendored_frameworks` pattern did not match any file.

Pods workspace available at `/var/folders/hz/7v68p4h95sb2ypb614xp20jc00r7sc/T/CocoaPods-Lint-20220523-21695-8rqgf2-FirebaseAnalytics/App.xcworkspace` for inspection.

[!] FirebaseAnalytics did not pass validation, due to 1 error.

[!] Skipping compilation with `xcodebuild` because target contains no sources.


[!] Skipping compilation with `xcodebuild` because target contains no sources.


[!] Skipping compilation with `xcodebuild` because target contains no sources.
```

Reply:
So, I think may be right: `pod lib lint` may not work with binary pods.

We spec lint GoogleAppMeasurement in CI and I expect we can do the same for this new spec.